src/audio/kpb.c

```diff
@@ -55,8 +55,8 @@ struct comp_data {
 	uint32_t source_period_bytes; /**< source number of period bytes */
 	uint32_t sink_period_bytes; /**< sink number of period bytes */
 	struct sof_kpb_config config;   /**< component configuration data */
-	struct comp_buffer *rt_sink; /**< real time sink (channel selector ) */
-	struct comp_buffer *cli_sink; /**< draining sink (client) */
+	struct comp_buffer *sel_sink; /**< real time sink (channel selector ) */
+	struct comp_buffer *host_sink; /**< draining sink (client) */
```

Comment:
good change, can you change the comment messages also? we usually use notion "real-time" for the host_sink? @lbetlej do you think so?

Reply:
@keyonjie, yes but "real-time" is confusing because in both kind of copies sel_sink/host_sink you deal with some sort of real time. When you copy to selector it is real time stream. When you copy to host in draining on demand mode (after normal draining is done) then you also copy real time stream - this time however directly to host bypassing selector.

Reply:
@keyonjie @mrajwa you have both been working on KWD recently so it's probably best to also have some documentation for this component in sof-docs that you both agree with :) This will also help the others understand the concepts and architecture of this component.

Reply:
Adjust comments accordingly.

Reply:
@tlauda I think comments are fine?

Reply:
@mrajwa I was thinking about changing `/**< draining sink (client) */` to `/**< draining sink (host) */`, but I guess it can stay this way.